lib/event-bus/src/EventBus.js

```diff
@@ -24,23 +24,31 @@ class EventBus {
         this._logger.trace({event}, 'Received event');
         const eventName = event.name;
 
-        const eventHandlers = this._handlers[eventName] || [];
+        const eventHandlers = [...this._handlers[eventName] || []];
 
         // allow global wildcards
-        if('#' in this._handlers) eventHandlers.push(this._handlers['#'][0]);
-        if('*' in this._handlers) eventHandlers.push(this._handlers['*'][0]);
+        if ('#' in this._handlers) {
+            eventHandlers.push(...this._handlers['#']);
+        }
+        if ('*' in this._handlers) {
+            eventHandlers.push(...this._handlers['*']);
+        }
 
         // allow second level wildcards
         const fragments = eventName.split('.');
-
```

Comment:
if(fragments.length < 2 && '#' in this._handlers) eventHandlers.push(this._handlers['#']);

Reply:
Suggestion: add the code above and remove lines 30-32 to make "#" behave more closely to rabbitmq standard